lib/client/Client.js

```diff
@@ -126,6 +126,22 @@ export default class Client {
     return this;
   }
 
+  /**
+   * Initializes the client with a list of custom mailbox roles.<br />
+   * Mailboxes having these custom roles will be preserved their role.
```

Comment:
s/Mailboxes having these custom roles will be preserved their role./Mailboxes having these custom roles will preserve their role./